libqtile/layout/xmonad.py

```diff
@@ -367,19 +379,19 @@ def configure(self, client, screen):
                 px,
                 margin=self.margin,
             )
-            client.unhide()
         else:
             # main client
             width = width_main - 2 * self.border_width
             client.place(
                 xpos + self.margin,
                 self.group.screen.dy + self.margin,
                 width - self.margin,
-                self.group.screen.dheight - 2 * self.border_width - 2 * self.margin,
+                (self.group.screen.dheight -
+                    2 * self.border_width - 2 * self.margin),
                 self.border_width,
                 px,
             )
-            client.unhide()
+
```

Comment:
Also, Travis pep8 checking doesn't like having 2 blank lines here. I feel like silly checks like this could be safely ignored...